### PR TITLE
Got <BV> to stop showing up in LMP deps

### PR DESCRIPTION
**Pull request by reecelikesramen**
Fix for #287 
* culprit is Black formatter adds space after # comment
* it doesn't always though because if the formatter raises exception, no format
* fixed by chaning `#<BV>` => `# <BV>` but for all tags

NOTE: this might not be the right way since without rerunning all the LMPs, the old source persists in the DB, so the old #<BV> tags won't get rendered right in the React app. Could instead enforce that # <BV> becomes #<BV> even after formatting, or strip out the delimiter before formatting and add it back in after.

### CODE REVIEW

**Comment by reecelikesramen** [comment]
Something else less important I noticed while working on this had to do with recognizing imports as dependencies. In `__testing__.py` one of my LMP's has an Optional[str] as a parameter type so it depends on `from typing import Optional`. However, in ell-studio that dependency resolves as `from typing import _UnionGenericAlias` probably due to how the lexical analysis works since Optional inherits from _UnionGenericAlias

**Comment by reecelikesramen** [comment]
Before:
image
After:
image

In `DiffRenderer.js` just filtering these out of the `code` and `previousCode`:
```js
function removeEllTags(code) {
  const ellTagRegex = /\s*#\s*<\/?(BV|BmV|LMP)>/g
  return code.replace(ellTagRegex, "")
}

code = removeEllTags(code)
previousCode = removeEllTags(previousCode)
```

File: ell-studio/src/components/source/DiffRenderer.js [hunk 1]
```diff
@@ -12,6 +12,14 @@ export function DiffRenderer({
 }) {
   if (!previousCode) return null;
 
+  function removeEllTags(code) {
+    const ellTagRegex = /\s*#\s*<\/?(BV|BmV|LMP)>/g;
+    return code.replace(ellTagRegex, '');
+  }
+
+  code = removeEllTags(code);
+  previousCode = removeEllTags(previousCode);
```
Comment (MadcowD) [inline ell-studio/src/components/source/DiffRenderer.js:21]: i think we dont want to remvoe tags. but if this is only for the diff renderer i guess thats fine ;)

**Comment by MadcowD** [comment]
Okay this is great it works! :) nvm disregard prev comments id like to merge

**Pull request merged**